app/firebase/functions/src/platforms/mastodon/mastodon.service.ts

```diff
@@ -205,26 +200,30 @@ export class MastodonService
     >
   ): Promise<FetchedResult<MastodonThread>> {
     if (DEBUG) logger.debug('fetch', { params, credentials }, DEBUG_PREFIX);
-    const { server, accountId } = parseMastodonUserId(user_id);
+    const { server, localUsername } = parseMastodonAccountURL(user_id);
     const client = this.getClient(server, credentials?.read);
 
+    const account = await client.v1.accounts.lookup({
```

Comment:
@pepoospina I realized we need the transaction manager to be able to get an account from the database. Not sure what the best way to make that available in the platform services